src/mito2/src/config.rs

```diff
@@ -83,6 +87,7 @@ impl Default for MitoConfig {
             global_write_buffer_reject_size: ReadableSize::gb(2),
             sst_meta_cache_size: ReadableSize::mb(128),
             vector_cache_size: ReadableSize::mb(512),
+            page_cache_size: ReadableSize::gb(2),
```

Comment:
I think the default value must not be so large, considering that `vector_cache_size` is 512Mib by default.